content/correspondence/reference/API endpoints/initialize/_index.en.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Initialize
+linktitle: Initialize
+description: Endpoint for initializing a correspondence.
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence
+
+## Authentication
+
+Use the endpoints in the authentication folder to authenticate before initializing.
```

Comment:
Most users wont connect to our API using our postman collection, but from their own API. 
Should describe that they need a Altinn token with correct scope. Can link to setup page if needed

Reply:
Describe which scopes is needed. Can check how other API's have written it